README.md

```diff
@@ -41,20 +41,29 @@
 - [GitHub Writes Code](https://github.com/JohnPhamous/GitHub-Writes-Code)
 - [HBD](https://github.com/vinitshahdeo/HBD)
 - [Hack-Day](https://github.com/lugnitdgp/Hack-Day)
-- [Hacktoberfest Animations](https://github.com/NiallEccles/Hacktoberfest-animations)
+- [Hacktoberfest](https://github.com/shivaylamba/Hacktoberfest)
```

Comment:
It's kind off conflicting with Hacktoberfest animations since you've added more than one repository in a single pull request, can you fix it rebasing or re adding hacktoberfest animations too?

Reply:
Sure! I will update the changes requested.